covid analyzer/allcalculations.py

```diff
@@ -0,0 +1,65 @@
+
+import FileReader
+from collections import Counter
+
+
+def recovered_over_total(args_country):
+    data1 = FileReader.FileReader('covid_cases_stats.csv')
+    country_input = args_country
+    print(country_input)
+    for row in data1:
+        if row['country'] == country_input:
+            print("%.2f" % (int(row['total_recovered'])/int(row['total_cases'])))
+
+
+def measures_death_rate(args_measure):
+    data1 = FileReader.FileReader('covid_cases_stats.csv')
+    data2 = FileReader.FileReader('covid_safety_measures.csv')
```

Comment:
There should be no file reading in this module. The FileReader should only be used by the main file.